aws-ssm-association/src/main/java/com/amazonaws/ssm/association/util/ResourceHandlerRequestToStringConverter.java

```diff
@@ -0,0 +1,56 @@
+package com.amazonaws.ssm.association.util;
+
+import com.amazonaws.ssm.association.ResourceModel;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+/**
+ * Used for safe printing of {@link ResourceHandlerRequest<ResourceModel>} requests.
+ */
+public class ResourceHandlerRequestToStringConverter {
```

Comment:
Renaming this to reflect that it hides private fields might make the intent more clear